testing/cli/test_sct_run_batch.py

```diff
@@ -61,70 +59,73 @@ def test_config_extra_value_warning():
         config.write(dedent(cnf_text))
         config.flush()
 
-        with pytest.warns(UserWarning, match='unknowable'):
-            # I'm not sure how to check that argparse is printing the right error here, but I trust
-            with pytest.raises(FileNotFoundError):
-                sct_run_batch.main(['-c', config.name, '-script', 'script'])
+    with pytest.warns(UserWarning, match='unknowable'):
+        # I'm not sure how to check that argparse is printing the right error here, but I trust
+        with pytest.raises(FileNotFoundError):
+            sct_run_batch.main(['-c', config_path, '-script', 'script'])
 
 
-def test_only_one_include():
-    # This is formatted strangely because pep8 says so.
-    with \
-        TemporaryDirectory() as data,\
-            TemporaryDirectory() as out:
-        with pytest.raises(AssertionError, match='Only one'):
-            sct_run_batch.main(['-include', 'arg', '-include-list',
-                                'arg2', '-path-data', data, '-path-out', out
-                                , '-script', out])
+def test_only_one_include(tmp_path):
+    data = tmp_path / 'data'
+    data.mkdir()
+    out = tmp_path / 'out'
+    with pytest.raises(AssertionError, match='Only one'):
+        sct_run_batch.main(['-include', 'arg', '-include-list', 'arg2',
+                            '-path-data', str(data), '-path-out', str(out), '-script', str(out)])
```

Comment:
It would be nice to use your `dummy_script` fixture here instead of `'-script', str(out)`

Reply:
Definitely agree!